### PR TITLE
fix(layout): ltr and rtl layout of main app elements

### PR DESCRIPTION
**Pull request by alebinson**

### CODE REVIEW

File: src/common/config/index.ts [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+    SERVERTYPE: 'geoserver',
+    TRANSITION: 0.5,
+  },
+  XYZ_LAYER: {
+    ATTRIBUTIONS: `Tiles © <a href="${MAP_SERVER}">GEE</a>`,
+    URL: `${MAP_SERVER}/${PUBLISH_POINT}/query?request=${REQUEST}&channel=${CHANNEL}&version=${VERSION}&x={x}&y={y}&z={z}`,
+  },
+};
+
+export default EXPORTER_CONFIG;
```
Comment (asafMasa) [inline src/common/config/index.ts:56]: end of line

File: src/common/config/index.ts [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+const MAP_SERVER = (window as any)._env_.MAP_SERVER;
+const PUBLISH_POINT = (window as any)._env_.PUBLISH_POINT;
+const CHANNEL = (window as any)._env_.CHANNEL;
+const VERSION = (window as any)._env_.VERSION;
+const REQUEST = (window as any)._env_.REQUEST;
+const SERVICE_PROTOCOL = (window as any)._env_.SERVICE_PROTOCOL;
+const SERVICE_NAME = (window as any)._env_.SERVICE_NAME;
+const ACTIVE_LAYER = (window as any)._env_.ACTIVE_LAYER;
+
+const EXPORTER_CONFIG = {
```
Comment (asafMasa) [inline src/common/config/index.ts:11]: use confd

Comment (asafMasa) [reply src/common/config/index.ts:11]: git ignore file?

File: src/discrete-layer/components/map-container/map-container.tsx [hunk 3]
```diff
@@ -1,21 +1,25 @@
...
+import { useTheme } from '@map-colonies/react-core';
 import { PolygonSelectionUi } from './polygon-selection-ui';
 import { MapWrapper } from './map-wrapper';
 import './map-container.css';
 
 export interface MapContainerProps {
   handlePolygonSelected: (polygon: Polygon) => void;
   handlePolygonReset: () => void;
+  handleOtherDrawers: () => void;
+  mapActionsWidth: string,
```
Comment (asafMasa) [inline src/discrete-layer/components/map-container/map-container.tsx:13]: ; instead of ,

**Review by asafMasa — CHANGES_REQUESTED** [review]

File: src/common/config/index.ts (hunk 1 above)
Comment (alebinson) [reply src/common/config/index.ts:56]: Done

File: src/common/config/index.ts (hunk 2 above)
Comment (alebinson) [reply src/common/config/index.ts:11]: Relevant **window.__env__** props are come from confd - it is a confd usage.
confd generates js file which downloaded and run on client.
This file included in index.html (same principals as in image exporter).

In .gitignore this file **public/env-config.js** - confd generated.

File: src/discrete-layer/components/map-container/map-container.tsx (hunk 3 above)
Comment (alebinson) [reply src/discrete-layer/components/map-container/map-container.tsx:13]: Nice one :)
DONE

**Review by asafMasa — APPROVED** [review]

**Pull request merged**